// strapi-admin/admin/src/components/LeftMenu/LeftMenuHeader/Wrapper.js
import styled from 'styled-components';
import PropTypes from 'prop-types';

import Logo from '../../../assets/images/logo-strapi.png';

const Wrapper = styled.div`
  background-color: #007eff;
  padding-left: 2rem;
  height: ${props => props.theme.main.sizes.leftMenu.height};

  .leftMenuHeaderLink {
    &:hover {
      text-decoration: none;
    }
  }

  .projectName {
    width: 100%;
    font-size: 1.5em;
    
    font-weight: 600;
    padding: 0px 30px;
    background-size: 30px;
    font-family: 'Poppins', sans-serif;
    color: white;
    align-items: center;
    display: flex;
    height: 6rem;

    background-image: url(https://noansa.vercel.app/icon/feather.png);
    background-repeat: no-repeat;
    background-position: left center;
    background-size: 30px;
  }
`;

Wrapper.defaultProps = {
  theme: {
    main: {
      colors: {
        leftMenu: {},
      },
      sizes: {
        header: {},
        leftMenu: {},
      },
    },
  },
};

Wrapper.propTypes = {
  theme: PropTypes.object,
};

export default Wrapper;
